### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

File: src/test_util.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+use std::{cmp::PartialEq, fmt::Debug};
```
Comment (rvolosatovs) [inline src/test_util.rs:1]: Maybe this file should reside within `tests` subdirectory in the root?

File: src/tabs/status.rs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+use super::prelude::*;
```
Comment (rvolosatovs) [inline src/tabs/status.rs:1]: If we're refactoring, how about adding spacing between import groups like `cargo fmt` would do by default?

File: src/tabs/tests.rs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+use super::*;
```
Comment (rvolosatovs) [inline src/tabs/tests.rs:1]: how about putting this into `/tests/tabs.rs` instead?

File: src/tabs/status.rs (hunk 2 above)
Comment (flosse) [reply src/tabs/status.rs:1]: This is a personal preference. I always use `cargo fmt` but without a special config and no spaces were added.
So maybe `cargo fmt` on a nightly channel behaves different?

File: src/tabs/tests.rs (hunk 3 above)
Comment (flosse) [reply src/tabs/tests.rs:1]: This is also a personal taste. I prefer to have most tests close to the actual code.

File: src/test_util.rs (hunk 1 above)
Comment (flosse) [reply src/test_util.rs]: ok, I moved now all tests into the integrations tests folder.

**Pull request merged**